### PR TITLE
add service

### PR DESCRIPTION
**Pull request by IvanAleinikau**

### CODE REVIEW

File: lib/data/repositories/saved_news_repository.dart [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        );
+      },
+      version: 1,
+    );
+  }
+  
+  Future<int> insertSavedNews(SavedNews savedNews) async {
+    int result = 0;
+    final Database db = await initializeDB();
+    result = await db.insert('news', savedNews.toJson());
```
Comment (Platton96) [inline lib/data/repositories/saved_news_repository.dart:24]: add empty state

File: lib/data/repositories/saved_news_repository.dart [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import 'package:sqflite/sqflite.dart';
+import 'package:path/path.dart';
+
+class SavedNewsRepository {
+  Future<Database> initializeDB() async {
+    String path = await getDatabasesPath();
+    return openDatabase(
+      join(path, 'saved_news.db'),
+      onCreate: (database, version) async {
+        await database.execute(
```
Comment (Platton96) [inline lib/data/repositories/saved_news_repository.dart:13]: what is it?

File: lib/data/repositories/saved_news_repository.dart [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+          'CREATE TABLE news(id INTEGER PRIMARY KEY AUTOINCREMENT,user TEXT,title TEXT,text TEXT,date TEXT)',
+        );
+      },
+      version: 1,
+    );
+  }
+  
+  Future<int> insertSavedNews(SavedNews savedNews) async {
+    int result = 0;
+    final Database db = await initializeDB();
```
Comment (Platton96) [inline lib/data/repositories/saved_news_repository.dart:23]: don't like work with db

File: lib/data/repositories/saved_news_repository.dart (hunk 2 above)
Comment (IvanAleinikau) [reply lib/data/repositories/saved_news_repository.dart:13]: when creating the db, create the table

File: lib/data/repositories/saved_news_repository.dart (hunk 1 above)
Comment (IvanAleinikau) [reply lib/data/repositories/saved_news_repository.dart:24]: ??

File: lib/data/repositories/saved_news_repository.dart (hunk 3 above)
Comment (IvanAleinikau) [reply lib/data/repositories/saved_news_repository.dart]: rename and add late

**Review by Platton96 — APPROVED** [review]

**Pull request merged**